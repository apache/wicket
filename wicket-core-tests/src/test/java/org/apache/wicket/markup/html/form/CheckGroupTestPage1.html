<html>
<head><title>CheckGroupTestPage1</title></head>
<body>
    <!--  In addition test that chars are not converted from upper to lower and vice versa -->
	<FORM wicket:id="form">
		<span wicket:id="group">
			<input type="checkbox" wicket:id="check1">check1</input>
			<span wicket:id="container">
				<input type="checkbox" wicket:id="check2">check2</input>
			</span>
		</span>
	</FORM>
</body>
</html>
