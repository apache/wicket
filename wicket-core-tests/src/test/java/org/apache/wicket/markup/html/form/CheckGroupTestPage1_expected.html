<html>
<head><title>CheckGroupTestPage1</title></head>
<body>
    <!--  In addition test that chars are not converted from upper to lower and vice versa -->
	<FORM wicket:id="form" id="form1" method="post" action="./org.apache.wicket.markup.html.form.CheckGroupTestPage1?0-1.-form"><div id="form1_hf_0" hidden="" class="hidden-fields"></div>
		
			<input type="checkbox" wicket:id="check1" id="check12" name="group" value="check0">check1</input>
			<span wicket:id="container">
				<input type="checkbox" wicket:id="check2" id="check23" name="group" value="check1">check2</input>
			</span>
		
	</FORM>
</body>
</html>
